cookbooks/view_pings_cep.md

```diff
@@ -0,0 +1,57 @@
+# See My Pings
+
+So you want to see what you're sending the telemetry pipeline, huh? Well follow these steps and we'll have you reading some JSON in no time.
+
+Note that some of these steps were stolen from the [Creating a Real-Time Analysis Plugin Cookbook](realtime_analysis_plugin.md).
+
+## Steps to Create a Viewing Output
+
+1. Get your client_id from whatever product you're using. For desktop, it's available in `about:telemetry`.
+
+2. Go to the CEP site: https://pipeline-cep.prod.mozaws.net/
+
+3. Login/Register using your Google `@mozilla.com` account
+
+4. Click on the `Analysis Plugin Deployment` tab
+
+5. Under "Heka Analysis Plugin Configuration", put the following config:
```

Comment:
nit: most things are referring to UI sections with quotes, but you're using backticks above in #4 - please be consistent with one or the other (I prefer quotes).